config/deploy/staging.rb

```diff
@@ -1 +1,3 @@
 server 'api-staging.goodcity.hk:59207', user: 'deployer', roles: %w{web app db}
+
+set :branch, :'GCW-1895-twilio-upgrade'
```

Comment:
@abulsayyad123 probably want to remove this now?

Reply:
Removed.